frontend/talentsearch/src/js/components/search/deprecated/SearchForm.tsx

```diff
@@ -184,7 +184,7 @@ export const SearchForm: React.FunctionComponent<SearchFormProps> = ({
     [cmoAssets, locale, intl],
   );
 
-  const operationalRequirementsSubset = [
+  const operationalRequirementsSubsetV1 = [
     OperationalRequirement.ShiftWork,
     OperationalRequirement.WorkWeekends,
     OperationalRequirement.OvertimeScheduled,
```

Comment:
Ah, I hadn't realized there was an even earlier subset -- sorry if that confused my suggestion :)

Reply:
no worries
all good to merge?